Editor/Window/GameLiftRequestAdapter.cs

```diff
@@ -0,0 +1,257 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Amazon.GameLift;
+using Amazon.GameLift.Model;
+using AmazonGameLift.Editor;
+using AmazonGameLiftPlugin.Core.ApiGatewayManagement;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Window
+{
+    public interface IAmazonGameLiftClientFactory
+    {
+        IAmazonGameLiftClientWrapper Get(string profile);
+    }
+
+    public class AmazonGameLiftClientFactory : IAmazonGameLiftClientFactory
```

Comment:
This interface/class shouldn't be declared in this file

Reply:
Moved